Source/Core/VideoBackends/OGL/TextureCache.cpp

```diff
@@ -294,7 +294,7 @@ void TextureCache::CompileShaders()
 		"\n"
 		"void main(){\n"
 		"	vec4 texcol = texture(samp9, vec3(f_uv0.xy, %s));\n"
-		"	int depth = clamp(int(texcol.x * 16777216.0), 0, 0xFFFFFF);\n"
+		"	int depth = int(floor(texcol.x * 16777216.0));\n"
```

Comment:
This may overflow so that depth = 0x1000000. Either multiply by 0xFFFFFF or clamp.